cpp/src/hash/hashing.cu

```diff
@@ -718,6 +720,60 @@ std::unique_ptr<column> md5_hash(table_view const& input,
                              mr);
 }
 
+std::unique_ptr<column> serial_murmur_hash3_32(table_view const& input,
+                                               uint32_t seed,
+                                               cudaStream_t stream,
```

Comment:
Since it looks like other parts of this code are using rmm::cuda_stream_view, should probably plumb that through all the way.

Edit: actually, looks like this function is already declared using rmm::cuda_stream_view, just not defined that way
```suggestion
                                               rmm::cuda_stream_view stream,
```

Reply:
Yes there should be no more references to `cudaStream_t` in libcudf (after #6744 is merged, but many functions are already converted). Also make sure you always order stream before mr, and try to follow the header grouping and ordering rules (separate common header groups with a blank line, order them from "nearest" to "farthest", and let clang-format sort them within their groups) since I spent a lot of time cleaning that up throughout the library.